src/lib.rs

```diff
@@ -22,23 +23,31 @@ pub struct Matches {
 
 impl fmt::Debug for Matches {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "{} has {} matche(s)", self.path, self.lines.len())
+        write!(f, "{} has {} match(es)", self.path, self.lines.len())
     }
 }
 
+#[tracing::instrument]
```

Comment:
This proc-macro annotates the method for `tracing`, helping to capture parameters and some extra info